hieradata/common.yaml

```diff
@@ -781,6 +781,14 @@ govuk_jenkins::config::user_permissions:
 
 govuk::node::s_logging::apt_mirror_hostname: "%{hiera('apt_mirror_hostname')}"
 
+govuk_mtail::enabled: true
+govuk_mtail::extra_args: "-log_dir /var/log/mtail -graphite_prefix %{::fqdn_metrics}."
+govuk_mtail::graphite_host_port: 'graphite:2003'
```

Comment:
is this correct for Carrenza? should it be `graphite.cluster`?

Reply:
Updated